draft-ietf-rats-corim.md

```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
 
 All of the validated and potentially useful tags are loaded into the Appraisal Context.
+This concludes the Appraisal Context initialisation phase.
 
 [^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/96
 
-This concludes the initialisation phase.
-
-## Evidence Collection
+## Evidence Collection {#sec-ev-coll}
 
-In the Evidence collection phase the Verifier communicates with attesters to
-collect Evidence.
+During the Evidence collection phase, the Verifier communicates with attesters to gather Evidence.
+The first part of this phase does not involve any cryptographic validation.
+This means that Verifiers can use untrusted code for their initial Evidence collection.
+The Evidence collection generates protocol-specific data and transcripts that are used as input to appraisal by the Verifier.
 
-The first part of the Evidence collection phase does not perform any
-cryptographic validation.
-This allows Verifiers to use untrusted code for their initial Evidence collection.
-
-The results of the evidence collection are protocol specific data and transcripts
-which are used as input to appraisal by the Verifier.
+## Evidence Appraisal {#sec-ev-appraisal}
 
 ### Cryptographic validation of Evidence {#sec-crypto-validate-evidence}
 
-If the authenticity of Evidence is secured by a cryptographic mechanism such as
-a signature, the first step in the Evidence Appraisal is to perform
-cryptographic validation of the Evidence.
-
-The exact cryptographic signature validation mechanics depend on the specific
-Evidence collection protocol.
-
-For example:
-In DICE, a proof of liveness is performed on the final key in the certificate
-chain.
-If this passes then a suitable certification path anchored on a trusted root
-certificate is looked up -- e.g., based on linking information obtained from
-the DeviceID certificate (see Section 9.2.1 of {{DICE.Layer}}) --
-in the Appraisal Context.  If found, then usual X.509 certificate validation
-is performed.
-In PSA, the verification public key is looked up in the appraisal context using
-the `ueid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
+If Evidence is cryptographically signed, its validation is the first step in Appraisal.
+
+The way cryptographic signature validation works depends on the specific Evidence collection protocol being used.
+For instance, in DICE, a proof of liveness is carried out on the final key in the certificate chain.
+If this is successful, a suitable certification path is looked up in the Appraisal Context, based on linking information obtained from the DeviceID certificate (see Section 9.2.1 of {{DICE.Layer}}).
+If a trusted root certificate is found, the usual X.509 certificate validation is performed.
+On the other hand, in PSA, the verification public key is looked up in the appraisal context using the `ueid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
```

Comment:
Should this text be moved to the PSA profile?

Reply:
I am not against dropping this.  However, it provides a concrete example of how different profiles may plug in their logic.

Reply:
Lines 1485-1487 appear to apply to a DICE example. Line 1488 doesn't begin with enough descriptive text to highlight it as a second example, vs. an extension to a DICE example. As a second example, it needs more development to make it understandable. I didn't want to spend too much time in this document reviewing content that applies to a different audience. Possibly, a reference to the psa document is sufficient for an interested reader to go there to obtain the well-described example?